docs/production-deployment/cloud/introduction/sla.mdx

```diff
@@ -37,16 +43,18 @@ Errors that aren't counted against the SLA include the following:
 - `PermissionDenied`
 - `QueryFailed`
 - `RetryReplication`
-- `ShardOwnershipLost`
 - `StickyWorkerUnavailable`
 - `TaskAlreadyStarted`
 - `Throttling (resources exhausted; triggers retry)`
 - `WorkflowExecutionAlreadyStarted`
 - `WorkflowNotReady`
 
-Our internal alerting system is based on our service level objectives (SLO) for all errors, not just errors that count against the SLA.
+Our internal alerting system is based on a [service level objective](https://en.wikipedia.org/wiki/Service-level_objective) (SLO) for all errors, not just errors that count against the SLA.
```

Comment:
capitalize Service Level Objective? Unsure.

Reply:
![image](https://github.com/temporalio/documentation/assets/156466156/fbe56b18-382c-49b1-9f28-3ae07f86d2de)

Reply:
Thank you for the sanity check. I appreciate this kind of feedback.